ansible.cfg

```diff
@@ -362,15 +362,15 @@
 #   * smart = try sftp and then try scp [default]
 #   * True = use scp only
 #   * False = use sftp only
-#scp_if_ssh = smart
+scp_if_ssh = True
 
 # Control the mechanism for transferring files (new)
 # If set, this will override the scp_if_ssh option
 #   * sftp  = use sftp to transfer files
 #   * scp   = use scp to transfer files
 #   * piped = use 'dd' over SSH to transfer files
 #   * smart = try sftp, scp, and piped, in that order [default]
-#transfer_method = smart
+transfer_method = scp
```

Comment:
Problem with working with non-ssh/scp methods is that jumping hosts becomes really hard (agent issues) and custom ssh options become useless so I always have opted for scp/ssh. Also, I think that sftp requires sftp config on the server for it to work right so it seems just easier to default everything to ssh/scp. I'd agree that `scp_if_ssh` might not be required but I think some versions defaulted to Paramiko on systems without it specified previously but it can probably be removed if the config comments are correct.